packages/labs/gen-manifest/src/index.ts

```diff
@@ -197,14 +202,77 @@ const convertClassDeclaration = (
       superClass ? convertReference(superClass) : undefined
     ),
     // mixins: [], // TODO
-    // members: [
-    //   // TODO: ClassField
-    //   // TODO: ClassMethod
-    // ],
+    ...useIfNotEmpty('members', [
```

Comment:
Similar comment about `undefined` being ok here, so we don't need spread.

Reply:
Done